CHANGELOG.md

```diff
@@ -34,6 +34,8 @@ Here is a template for new release sections
 - `COST_REPLACEMENT` is now a parameter that is included in output cost matrix (#613)
 - Improved `Code.rst` for RTD code documentation (#704)
 - All `.py` files to add a module description for RTD on top (#704)
+- Changed `C0.energyStorage()` for timeseries in storage parameters (hotfix) (#720)
+- Input files and benchmark test `test_benchmark_special_features.Test_Parameter_Parsing()`: Now also including timeseries in a storage component ()
```

Comment:
Hi @MaGering !

> "In your PR, please also add the correct number for the benchmark tests changelog entry."

Refers to the lines above. I added changelog entries for what you changed, where I also put your PR as origin (#720). Line 38 is what I changed in this PR (#723). As I can not know what the PR number is when I am working on my lokal branch, I couldnt assign the number to the changelog entry. Please add the number in the brackets ;)

Reply:
You can change this either locally, or by opening the file view of the changelog in github, making a manual change, and submitting it.

Reply:
So I will do the changes in my [PR](https://github.com/rl-institut/multi-vector-simulator/pull/720), correct?